widget/playground/src/components/ExportConfigModal/ExportConfigModal.styles.tsx

```diff
@@ -47,17 +48,28 @@ export const StyledButton = styled(Button, {
     },
     type: {
       secondary: {
+        zIndex: 10,
         color: '$background',
-        backgroundColor: '$secondary500',
+        backgroundColor: 'transparent',
         '&:hover, &:focus': {
           color: '$background',
-          backgroundColor: '$secondary500',
+          backgroundColor: 'transparent',
         },
       },
     },
   },
 });
 
+export const BackdropTab = styled('div', {
+  width: '362px',
+  height: '$40',
+  backgroundColor: '$secondary500',
+  position: 'absolute',
+  borderRadius: '$xm',
+  inset: 0,
+  transition: 'transform 0.5s cubic-bezier(0, 0, 0.86, 1.2)',
```

Comment:
NIT: Considering that it is out of the container, I think a number smaller than one is better for the fourth parameter.

Reply:
changed.